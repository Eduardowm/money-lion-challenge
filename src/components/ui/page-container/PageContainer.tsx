import { PropsWithChildren } from 'react'

function PageContainer({ children }: PropsWithChildren) {
  return <div className="flex max-w-[600px] w-full mx-auto my-5 px-2">{children}</div>
}
export default PageContainer
