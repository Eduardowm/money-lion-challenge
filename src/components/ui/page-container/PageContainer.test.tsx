import { render } from '@testing-library/react'

import PageContainer from './PageContainer'

describe('PageContainer', () => {
  it('renders as expected', () => {
    const { container } = render(<PageContainer>This is the page container content</PageContainer>)

    expect(container).toMatchSnapshot()
  })
})
