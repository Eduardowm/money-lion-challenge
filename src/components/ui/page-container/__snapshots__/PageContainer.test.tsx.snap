// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContainer renders as expected 1`] = `
<div>
  <div
    class="flex max-w-[600px] w-full mx-auto my-5 px-2"
  >
    This is the page container content
  </div>
</div>
`;
